ush/config_defaults.sh

```diff
@@ -385,12 +397,26 @@ WFLOW_LAUNCH_LOG_FN="log.launch_FV3LAM_wflow"
 # FCST_LEN_HRS:
 # The length of each forecast, in integer hours.
 #
+# FCST_LEN_HRS_CYCLES:
+# The length of forecast for each cycle, in integer hours.
+# When it empty, all forecast will be FCST_LEN_HRS
+#
+# DA_CYCLE_INTERV:
+# Data assimilation cycle interval, in integer hours for now.
+#
+# RESTART_INTERVAL:
+# Set up frequenency or list of the forecast hours that FV3 should
+# generate the restart files. The numbers need to match with DA_CYCLE_INTERV.
```

Comment:
What does it mean to "match with DA_CYCLE_INTERV"?  To be multiples of? What happens when RESTART_INTERVAL=3,6?

Reply:
RESTART_INTERVAL  is a parameter for model forecast. But I put "match" here to remind we need to consider data assimilation cycling interval when setup it for data cycling. For now, I don't want to put a clear formula for the relation.
 "RESTART_INTERVAL=3,6" will make hourly cycling use the background from previous 3-h cycle instead of using previous 1-h cycle.

Reply:
The comment on how to set this value needs clearer direction. I still am unclear on why it's a list, and what the items of the list should correspond to.

Reply:
It's also a comma separated list here, and a space separated list in the config file for dev2. Please indicate which should be used.

Reply:
Both format work and they have different means.
The list is to tell model which forecast hour it should write out the restart files. We want 1-6 forecast hour to have restart files for hourly cycling. The 1-h forecast is the background and 2-6 is fallback background if previous cycle crashed. The GSI has the functions to look back based on the cycling intervals. I don't think we need to put those details in the code. But we can certainly discuss those DA setup and fallback solutions off-line.

Reply:
I deleted "The numbers need to match with DA_CYCLE_INTERV". The current explanation for this parameter is clear on what this for. How to configure a cycling system should not be explained here.